src/data/skill-desc.csv

```diff
@@ -412,6 +414,8 @@
 "レース終盤で3番手の時に負けそうに\nなると速度がわずかに上がる","<size=22>When you're in 3rd place at the end of the race and about to lose, your speed slightly increases</size>"
 "レースを冷静に運ぶことができると残り\n200mで実力発揮、速度が少し上がる","<size=22>After proceeding through the race calmly, your speed increases a little during the last 200m</size>"
 "最終直線で前の方で追いすがっていると\n速度がわずかに上がる","<size=22>Slightly increase speed in the last straight when chasing someone close to the lead</size>"
+"中盤に後ろの方で順位を下げた時に\n持久力をわずかに回復する","<size=22>During Middle Leg, recover slight stamina when losing position in the back of the pack</size>"
```

Comment:
"recover slight stamina" -> "recover stamina slightly" or "slightly recover stamina"

Reply:
Also I'm pretty sure other change_order_onetime>0 skills use "when overtaken", not "when losing position"

Reply:
i guess i can change that, they pretty much the same, i was just trying to match the japanese one